src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,144 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationServiceImpl registrationServiceImpl;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        registrationServiceImpl = new RegistrationServiceImpl();
+        user = new User();
+        Storage.people.clear();
+    }
+
+    @Test
+    void userIsValid_ok() {
+        User existingUser = new User();
+        existingUser.setLogin("differentLogin");
+        Storage.people.add(existingUser);
+        user.setAge(20);
+        user.setLogin("validLogin");
+        user.setPassword("validPas");
+        User registeredUser = registrationServiceImpl.register(user);
+        assertEquals(user, registeredUser);
+    }
```

Comment:
This test method is named `userIsValid_ok`, but it doesn't follow the convention `<methodUnderTest>_<state>_<expectedBehavior>`. Consider renaming it to `register_validUser_ok` to maintain consistency and clarity.